src/System.Buffers/src/PACKAGE.md

```diff
@@ -0,0 +1,40 @@
+## About
+
+<!-- A description of the package and where one can find more documentation -->
+
+Provides resource pooling of any type for performance-critical applications that allocate and deallocate objects frequently.
+
+## Key Features
+
+<!-- The key features of this package -->
+
+*
+
+## How to Use
+
+<!-- A compelling example on how to use this package with code, as well as any specific guidelines for when to use the package -->
+
+
```

Comment:
Are all these sections needed?

Some of these (like key features) are fairly redundant given the `about` or aren't really possible to provide trivial "how to use" for sections around, as they are advanced topics (we don't even have examples in the main docs, as they are primarily covered by blogs or other more in depth topics instead).

Reply:
Well, this is the same template we used in runtime.

Fill out the sections that you think makes sense for your assembly. If you find blogs that cover usage, link to those. What's important is that we describe the assembly clearly.